doc/modules/clustering.rst

```diff
@@ -278,6 +285,67 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+
+:class:`KMedoids` is related to the :class:`KMeans` algorithm. While
```

Comment:
You can think of both as minimizing distances to cluster centers, though in KMeans they can be arbitrary and in KMedoids they can only be data points. Though that allows KMedoids to work with arbitrary metrics.